palm/plugins/dbt/templates/containerize/entrypoint.sh.txt

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash -x
+# Entrypoint script for docker
+{% if package_manager == 'poetry' %}
+pip3 install poetry
+poetry config virtualenvs.create false
+poetry install
+{% endif %}
+{% if package_manager == 'pip3' %}
+pip3 install -r requirements.txt
+pip3 install pytest
+{% endif %}
+{% if package_manager == 'unknown' %}
```

Comment:
I don't think this is a possible case anymore since the PythonContainerizer prompts to create a reqirements.txt and will exit without one. This can be removed

Reply:
Noted!